Tasks/PublishSymbols/task.json

```diff
@@ -37,25 +37,23 @@
             "label": "Search Pattern",
             "defaultValue": "**\\bin\\**\\*.pdb",
             "required": false,
-            "helpMarkDown": "The pattern used to discover the pdb files to publish.",
-            "groupName": "advanced"
```

Comment:
any reason we remove this option from advanced group? since it's not required and has a defaultValue.